src/lib/Behat/BrowserContext/UserPreferencesContext.php

```diff
@@ -8,15 +8,34 @@
 
 use Behat\Behat\Context\Context;
 use Ibexa\AdminUi\Behat\Page\ChangePasswordPage;
+use Ibexa\AdminUi\Behat\Page\UserSettingsPage;
 
 class UserPreferencesContext implements Context
 {
-    /** @var \Ibexa\AdminUi\Behat\Page\ChangePasswordPage */
-    private $changePasswordPage;
+    private ChangePasswordPage $changePasswordPage;
 
-    public function __construct(ChangePasswordPage $changePasswordPage)
+    private UserSettingsPage $userSettingsPage;
+
+    public function __construct(ChangePasswordPage $changePasswordPage, UserSettingsPage $userSettingsPage)
     {
         $this->changePasswordPage = $changePasswordPage;
+        $this->userSettingsPage = $userSettingsPage;
+    }
+
+    /**
+     * @Given I switch to :tabName tab
```

Comment:
```suggestion
     * @Given I switch to :tabName tab in User Preferences
```

I'd add this so that we don't have to worry about clashing step definitions in the future (there can be only one Step with this definition).

We already have similar Steps:
```
I switch to :tab field group
I switch to :tab tab in Content structure
```